layouts/shortcodes/fileRel.html

```diff
@@ -0,0 +1,6 @@
+{{- $pathURL := .Get "pathURL" -}}
+{{- $name := .Get "name" -}}
+{{- $class := .Get "class" -}}
+{{- $style := .Get "style" -}}
```

Comment:
```suggestion
{{- $class := .Get "class" | default "" -}}
{{- $style := .Get "style" | default "" -}}
```

since `class` and `style` params are optional, give them a default of empty string